Tests/ScipioKitTests/DescriptionPackageTests.swift

```diff
@@ -65,17 +65,20 @@ final class DescriptionPackageTests: XCTestCase {
     }
 
     func testBuildProductsInCreateMode() throws {
-        let rootPath = fixturePath.appendingPathComponent("BinaryPackage")
```

Comment:
The existing test might aim to test whether it can extract binary targets correctly so it's better to remain the existing tests.

Could you make a new test case for this change and rename the existing tests to `testBuildProductsForBinaryPackage` or something?

Reply:
Sure, my reasoning was that `BinaryPackage` was already involved in the integration tests, unlike `TestingPackage`, so I felt safer replacing the former. Of course, keeping the previous test in place makes sense for extra peace of mind. Just fixed that.

Reply:
Let me know if you'd like to change the test name. I thought a package couldn't really be binary, unlike its targets (or wrapping them build products).